server/boot/index.js

```diff
@@ -26,9 +26,6 @@ function setup() {
 	// for nginx
 	app.enable( 'trust proxy' );
 
-	// template engine
-	app.set( 'view engine', 'pug' );
-
```

Comment:
In the long run, it'd be nice to set a rendering engine for JSX here, so we can use `res.render( /*...*/ )` instead of `res.send( renderJsx( /*...*/ ) );`. (Not necessarily in this PR however.)

Reply:
I didn't do this in the first place because it's not possible to stream with `render` afaik. Streaming is not included yet so this is a reasonable concern.